packages/ActionMenu/src/ActionMenu.tsx

```diff
@@ -34,6 +34,10 @@ export interface ActionMenuProps extends React.ComponentProps<"div"> {
     /** Whether or not the menu should use ReactPortal
    * to append to the body */
     disablePortal?: boolean;
+    /** Add a specific class to the dropdown */
+    dropdownClassName?: string;
+    /** Offset of the dropdown */
```

Comment:
in rem? pixel?

Reply:
it's in px, but I see your point :P should I update the comment to say this?